test/internet/test-dns.js

```diff
@@ -553,7 +553,7 @@ console.log('looking up nodejs.org...');
 
 const cares = process.binding('cares_wrap');
 const req = new cares.GetAddrInfoReqWrap();
-cares.getaddrinfo(req, 'nodejs.org', 4);
+cares.getaddrinfo(req, 'nodejs.org', 4, /* hints */ 0, /* verbatim */ true);
```

Comment:
You could compare results to `python -c "import socket ; a = socket.getaddrinfo('nodejs.org', None); print(a)"`